nexus/examples/config.toml

```diff
@@ -127,6 +127,7 @@ saga_recovery.period_secs = 600
 lookup_region_port.period_secs = 60
 region_snapshot_replacement_start.period_secs = 30
 region_snapshot_replacement_garbage_collection.period_secs = 30
+region_snapshot_replacement_step.period_secs = 30
```

Comment:
As far as I know, they usually do start at the same time! :) I think we're ok - each task only operates on requests that are in a certain state, and so are bounded in that sense from it being a thundering herd.

Reply:
I'm thinking about a sled being expunged and all 10 crucible zones and all the things
that could be on those 10 zones, etc.  Making sure we can handle an outage like that.

Reply:
Right now we only limit the number of operations that can happen at one time to a Volume, and don't currently limit otherwise.I'm not too concerned

Reply:
(hit comment too early!) I'm not too concerned, but it will be quite a bit of traffic to have all those reconciliations, live repairs, and clones going on... on second thought maybe I am concerned haha.

I'm not sure what to do other than to test it out and see though. We might be able to come up with an upper bound on the number of regions and snapshots maybe? Even if we do we can't know what the non-expunge load on the rack will be at any given time.

What do you think? Do you think some sort of limit is a good idea?

Reply:
It's hard to say if we should set a limit without knowing what the load a repair might generate.

But, that being said, it is probably better to put in a limit to make things slower but still
able to finish and then loosen that limit as we discover how much the rack can handle.
The alternative would be to just let repair take as many resources as it wants and, then
let things break if it's too much?  That second choice does not sound like something
I would want to explain to a customer :)

We don't have anything yet we can use as an overall guage of rack "busyness".  In additon,
we don't know the load an "average repair" generates even.  Maybe our first steps would
be to build the tools to determine what the current repair load is so we can measure
it's impact?